boggle-backend/boggle-lib/src/main/java/com/boggle/Node.java

```diff
@@ -0,0 +1,39 @@
+package com.boggle;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Node {
+
+
+    private Map<Character, Node> children = new HashMap<Character, Node>();
+
+    public Node(){
+    }
+
+    public Map<Character, Node> getChildren(){
+        return children;
+    }
+
+    public Boolean hasChildren(){
+        if (children.isEmpty()){
+            return false;
+        }
+      return true;
+    }
+
+    public boolean childHasLetter(char letter){
+        if (children.containsKey(letter)) {
```

Comment:
can you do return children.containsKey(letter) instead? that will automatic return the right boolean